acquire/acquire.py

```diff
@@ -859,6 +859,24 @@ def get_spec_additions(cls, target: Target, cli_args: argparse.Namespace) -> Ite
         return spec
 
 
+@register_module("--spse")
+class SPSE(Module):
```

Comment:
I can't really find much resources online referring to this as "SPSE", maybe just use `--sharepoint`? We also do `--exchange`.